### PR TITLE
[TSC Common] Common Abbreviations

### LINKED ISSUE #1803: [TSC Common] Common Abbreviations
Add common abbreviations

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #1803 

List of abbreviations of terms used in the Common and README documents.

PS although CNI is a Kuberntes related terms and used more in RA-2, it is sued without explanation in a "common" document

### CODE REVIEW

File: doc/common/abbreviations.md [hunk 1]
```diff
@@ -17,9 +17,60 @@
 
 | Term           | Description                           |
 |----------------|-------------                          |
-| CNTT           | Cloud iNfrastructure Telco Taskforce  |  
-| PRD            | Permanent Reference Document          |
+| AI             | Artificial Intelligence               |
+| API            | Application Programming Interface     |
+| CaaS           | Container as a Service                |
+| CI/CD          | Continuous Integration / Continuous Deployment |
+| CNF            | Cloud native Network Functio          |
```
Comment (tomkivlin) [inline doc/common/abbreviations.md:24]:
```suggestion
| CNF            | Cloud native Network Function          |
```

File: doc/common/abbreviations.md [hunk 2]
```diff
@@ -17,9 +17,60 @@
...
+| IO             | Input/Output                          |
+| IOMMU          | Input/Ouput Memory Management Unit    |
+| IP             | Internet Protocol                     |
+| IT             | Information Technology                |
+| K8s            | Kubernetes                            |
+| LAN            | Local Area Network                    |
+| MANO           | Management and Orchestration          |
+| NFV            | Network Funciton Virtualisation       |
+| NFVI           | Network Funciton Virtualisation Infrastructure |
+| NFVO           | Network Funciton Virtualisation Orchestrator |
```
Comment (tomkivlin) [inline doc/common/abbreviations.md:48]:
```suggestion
| NFV            | Network Function Virtualisation       |
| NFVI           | Network Function Virtualisation Infrastructure |
| NFVO           | Network Function Virtualisation Orchestrator |
```

File: doc/common/abbreviations.md [hunk 3]
```diff
@@ -17,9 +17,60 @@
...
+| ONAP           | Open Network Automation Platform      |
+| OPNFV          | Open Platform for NFV                 |
+| OVP            | OPNFV Verified Program                |
+| PCI-PT         | PCIe PassThrough                      |
+| PF             | Physical Function                     |
+| RA             | Reference Architecture                |
+| RC             | Reference Conformance                 |
+| RI             | Reference Implementation              |
 | RM             | Reference Model                       |
+| SDN            | Software Define Networking            |
```
Comment (tomkivlin) [inline doc/common/abbreviations.md:59]:
```suggestion
| SDN            | Software-Defined Networking            |
```

**Comment by walterkozlowski** [comment]
@pgoyal01 please address the above comments from @tomkivlin and I will approve and we will merge.

**Pull request merged**